### ISSUE TITLE
Feature Request: default on/off variations

### ISSUE THREAD

**Issue by bschaeffer** [terraform-providers/terraform-provider-launchdarkly#18]
Re: [Launched: Choose Your Default Feature Flag Variations][post], we would really love to have this in the provider. 

It is becoming somewhat of a footgun for us to be unable to set the default variations in terraform. I am aware that we can set the `fallthrough` but we really don't want to have to track those. We actually don't track any of our targeting rules, mainly just variations, and being able to set the defaults only when the flag is created (which it looks like this feature allows) without tracking changes to fallthrough defaults per env would be a huge win for us.

Is this possible? Is it supported by the API? Would you accept a PR?

[post]: https://launchdarkly.com/blog/launched-choose-your-default-feature-flag-variations/,

**Comment by bschaeffer** [comment]
TLDR; we just want to control the initial default value per env, but not track changes to the default values as devs alter them using different strategies across environments.

**Comment by ldhenry** [comment]
Hey @bschaeffer,

I agree this would be a great feature to add. I just double-checked and it is indeed supported by our API. I just created an internal ticket to add this feature and I will try to get it prioritized in the near future. With that being said, we would be more than happy to accept a PR!

Thanks for your feedback,
Henry

**Comment by ldhenry** [comment]
Hey @bschaeffer, we added `default_on_variation` and `default_off_variation` to `launchdarkly_feature_flag` in v1.3.0. When setting these up, keep in mind the defaults should be set to the default variation *value* (i.e. `true`, `false` for booleans).

Thanks again for your feedback!

Cheers,
Henry

**Comment by bschaeffer** [comment]
@ldhenry You are a legend! Lots of fixes recently thank you so much for getting these in.
